src/main/java/core/basesyntax/dao/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.DataConverter;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String COMMA_SEPARATOR = ",";
+    private static final int EXPECTED_PARTS_LENGTH = 3;
+    private static final int MINIMAL_QUANTITY = 1;
```

Comment:
```suggestion
    private static final int MINIMAL_QUANTITY = 0;
```
it is better not to enter such restrictions, 0 can be used to reset the balance to zero, for example